stable/influxdb/values.yaml

```diff
@@ -0,0 +1,146 @@
+## influxdb image version
+## ref: https://hub.docker.com/r/library/influxdb/tags/
+image:
+  repo: "influxdb"
+  tag: "1.1.0-alpine"
+  pullPolicy: IfNotPresent
+
+## Specify a service type
+## NodePort is default
+## ref: http://kubernetes.io/docs/user-guide/services/
+service:
+  type: NodePort
+
+## Persist data to a persitent volume
+persistence:
+  enabled: false
+  storageClass: generic
+  accessMode: ReadWriteOnce
+  size: 8Gi
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources:
+  requests:
+    memory: 256Mi
+    cpu: 0.1
+  limits:
```

Comment:
I've done a quite a bit of testing on InfluxDB. That limit is a good general configuration for a medium to high load installation => https://docs.influxdata.com/influxdb/v1.1/guides/hardware_sizing/#high-load-recommendations